modules/meb_api/lib/authentication_token_service.rb

```diff
@@ -6,9 +6,9 @@ class AuthenticationTokenService
     TYP = 'JWT'
     KID = 'vanotify'
     SIGNING_KEY = Settings.dgi.jwt.private_key_path
-    RSA_PRIVATE = OpenSSL::PKey::RSA.new(File.read(SIGNING_KEY))
+    RSA_PRIVATE = OpenSSL::PKey::RSA.new(File.read(SIGNING_KEY)) if File.exist?(SIGNING_KEY)
```

Comment:
Now that the files are defined in Settings, you can remove `if File.exist?` here